tests/assertions/not-found.ts

```diff
@@ -9,7 +9,7 @@ export default function notFound<Context extends TestContext>(
 ) {
     const result = !(typeof subject === 'string' ? context.element.querySelector(subject) : subject);
     const actual = result ? 'not found' : 'found';
-    const expected = 'found';
+    const expected = 'not found';
```

Comment:
This is a little confusing, but, because this is negative assertion (see next line), it is actually correct for "expected" to be `'found'`, e.g.
 
<img width="245" alt="screen shot 2018-07-18 at 12 38 58 pm" src="https://user-images.githubusercontent.com/348630/42895359-add2bb7c-8a87-11e8-958f-561ac2f982d7.png">

Maybe a comment would be good to make this clear?

Reply:
Hmm, testem doesn't report this very nicely, though:

```
     expected: 'found'
       actual: 'found'
```

So maybe we should change it and not make it negative, since that only seems to work in-browser?